src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+    private String color = getColor();
+
+    public Circle(int radius, String color) {
+        this.radius = radius;
+        this.color = color;
+    }
+
+    @Override
+    public double areaCalculator() {
+        return (radius * radius * Math.PI);
```

Comment:
Magic numbers should be avoided. In this case, 'Math.PI' is fine because it's a well-known constant, but if there were other numeric values, they should be defined as constants with meaningful names.